<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Roshambo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app.qrc">
    <normaloff>:/bmp/figures/icon.bmp</normaloff>:/bmp/figures/icon.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_layout" stretch="9,1">
     <item>
      <layout class="QHBoxLayout" name="img_layout" stretch="10,1,10">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7,1">
           <item>
            <widget class="QLabel" name="me_label_img">
             <property name="text">
              <string>imgP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="me_label">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>PLAYER:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="me_score">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="me_img">
           <property name="text">
            <string>img1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>vs.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,7,1">
           <item>
            <widget class="QLabel" name="opposite_label_img">
             <property name="text">
              <string>imgC</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="opposite_label">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>COMPUTER:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="opposite_score">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="opposite_img">
           <property name="text">
            <string>img2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="btn_layout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="rock_btn">
         <property name="font">
          <font>
           <family>Juice ITC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ROCK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scissors_btn">
         <property name="font">
          <font>
           <family>Juice ITC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>SCISSORS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="paper_btn">
         <property name="font">
          <font>
           <family>Juice ITC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>PAPER</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="menu_about_btn"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_game">
    <property name="title">
     <string>game</string>
    </property>
    <addaction name="menu_restart_btn"/>
    <addaction name="separator"/>
    <addaction name="menu_PVE_btn"/>
    <addaction name="separator"/>
    <addaction name="menu_cheat_btn"/>
   </widget>
   <addaction name="menu_game"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_about_btn">
   <property name="text">
    <string>About me</string>
   </property>
  </action>
  <action name="actionHow_to_play">
   <property name="text">
    <string>How to play</string>
   </property>
  </action>
  <action name="menu_help_btn">
   <property name="text">
    <string>How to play</string>
   </property>
  </action>
  <action name="menu_restart_btn">
   <property name="text">
    <string>restart</string>
   </property>
  </action>
  <action name="menu_PVE_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PVE</string>
   </property>
  </action>
  <action name="menu_cheat_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cheat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
